### PR TITLE
Data access tutorial

### PR DESCRIPTION
**Pull request by mikala-nsidc**
the website build through GitHub actions is failing, because the notebooks need to authenticate with Earthdata Login.  We're doing that authentication with earthaccess and so we could provide credentials to the notebook via the following environment variables
EARTHDATA_USERNAME, EARTHDATA_PASSWORD
could you set those as repository secrets?  do you already have a shared account for this?

### CODE REVIEW

**Comment by mikala-nsidc** [comment]
Adding the Data Access chapter to the SnowEx website-2024 Jupyter Book.

**Comment by scottyhq** [comment]
> EARTHDATA_USERNAME, EARTHDATA_PASSWORD
could you set those as repository secrets? do you already have a shared account for this?

ref: https://earthaccess.readthedocs.io/en/latest/howto/authenticate/#authenticate-with-earthdata-login

I just added them @mikala-nsidc! So the CI workflows will have access via a bot "uwhackweek" account

File: conda/README.md
Comment (scottyhq) [inline conda/README.md:1]: You can remove the conda/ folder from this PR (I'll do a separate update of the CI environment to match cryocloud)

Comment (mikala-nsidc) [reply conda/README.md:1]: thanks @scottyhq !  Just to clarify - should I revert my commit that contains the change requests to anything under conda/?  Or do you mean actually remove the Conda/ folder from my fork and push that change?  Thanks for any help!

Comment (scottyhq) [reply conda/README.md:1]: I meant revert your change or copy from the new main branch. I think there are different ways to do this. Maybe this?: 

```
git checkout main
git pull upstream main
gh pr checkout 11 
git merge main
git checkout HEAD -- conda/
git push
```

**Comment by mikala-nsidc** [comment]
@scottyhq I reverted the commit I made yesterday that requested changes to files in conda/.  Let me know if this works, or if I need to do something else to make the pull request mergeable. Thanks!

File: book/tutorials/Data_access/NSIDC_resources.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+## NASA National Snow and Ice Data Center Distributed Active Archive Center (NSIDC DAAC)
+
+[The National Snow and Ice Data Center](https://nsidc.org) provides over 1100 data sets covering the Earth's cryosphere and more, all of which are available to the public free of charge. Beyond providing these data, NSIDC creates tools for data access, supports data users, performs scientific research, and educates the public about the cryosphere. 
+
+<figure>
+<center>
+   <img width="80%" src='./images/DAAC_map_with_ECS.jpg'/>
+</center>
+</figure>
```
Comment (scottyhq) [inline book/tutorials/Data_access/NSIDC_resources.md:20]: This are not rendering on CryoCloud, could you please try this syntax https://mystmd.org/guide/figures#image-directive

**Comment by scottyhq** [comment]
After #13 if you merge the main branch again I think your notebooks will pick up the EARTHDATA credentials and successfully execute

File: book/tutorials/Data_access/NSIDC_resources.md (hunk 1 above)
Comment (mikala-nsidc) [reply book/tutorials/Data_access/NSIDC_resources.md:20]: Will do!

File: book/tutorials/Data_access/earthaccess_snowex.ipynb [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "downloaded_files = earthaccess.download(\n",
+    "    results,\n",
+    "    local_path='./Downloads',\n",
```
Comment (scottyhq) [inline book/tutorials/Data_access/earthaccess_snowex.ipynb:258]: Consider changing the download path to `/tmp` for tutorial data (https://docs.2i2c.org/user/topics/data/filesystem/#the-tmp-directory)

**Comment by JessicaS11** [comment]
@mikala-nsidc This is looking great. I would love to see your and Gail's hard work putting this content together recognized if you're comfortable adding your names to the notebooks and/or overview page.

**Comment by mikala-nsidc** [comment]
I forgot to add in my commit message that I also changed the format of the figures, so hopefully they will render in the book in cryocloud now.  Let me know, if not!

**Comment by mikala-nsidc** [comment]
That last force push was to overwrite the commits by egreckase and go back to where we were prior to those.

**Review by scottyhq — APPROVED** [review]

**Comment by mikala-nsidc** [comment]
I see those commits @scottyhq !  Thanks for adding those for me.

**Pull request merged**